www/docs/en/dev/guide/platforms/android/index.md

```diff
@@ -299,6 +299,10 @@ To sign an app, you need the following parameters:
 These parameters can be specified using the command line arguments above to
 the [Cordova CLI](../../../cordova-cli/index.html) `build` or `run` commands.
 
+__Note__: You should use double `--` to shield these platform-specific arguments, for example:
```

Comment:
Is "shield" the correct word here? I think something like "... to indicate that these are platform-specific arguments ..." is more descriptive.